### PR TITLE
Clio 37536 thfilter money

### PR DESCRIPTION
**Pull request by dtomalty**

### CODE REVIEW

File: public/json/customFields.json [hunk 1]
```diff
@@ -15,6 +15,11 @@
     "id": "4"
   },
   {
+    "name": "money",
```
Comment (JefCurtis) [inline public/json/customFields.json]: The description in the `CustomField` serializer is "currency". Should this match?

File: themis_components/thFilter/thCustomFilterRow.directive.coffee [hunk 2]
```diff
@@ -21,6 +21,11 @@ angular.module 'ThemisComponents'
         {name: ">=", value: ">="}
         {name: ">", value: ">"}
       ]
+      @moneyOperatorOptions = [
+        {name: "Less than", value: "<"}
```
Comment (JefCurtis) [inline themis_components/thFilter/thCustomFilterRow.directive.coffee]: What about `>=` or `<=`? Those are both available as filters in the API.

Comment (dtomalty) [inline themis_components/thFilter/thCustomFilterRow.directive.coffee]: This matches the existing money filter. If we want to change it it'd be as easy as adding to this array.

File: public/json/customFields.json (hunk 1 above)
Comment (dtomalty) [inline public/json/customFields.json]: Yep good catch- updated.

**Pull request merged**